### ISSUE TITLE
Is gradient clipping supported?

### ISSUE THREAD

**Issue by gpengzhi** [facebookresearch/llama-recipes#277]
### 🚀 The feature, motivation and pitch

Does `llama-recipes` support `gradient clipping`?

I added an additional loss term to the original cross entropy loss and found that gradients (sometimes) become `NaN` after some iterations.

I could help add this feature if needed.

**Comment by HamidShojanazeri** [comment]
@gpengzhi thanks for the your interest sure, would be happy to review your PR and help to merge.
